### DISCUSSION TITLE
Cannot create the argocd k8s services.

### DISCUSSION THREAD

**Discussion by shvamabps** [argoproj/argo-cd#11550] [General]
![image](https://user-images.githubusercontent.com/37547983/205487054-6a0f6489-9083-4992-a56b-6f511915e6d8.png)
I don't know how I can do it correctly.
I have already created the namespace.

**Comment by kostis-codefresh** [comment]
This error seems to be unrelated to ArgoCD and instead has to do with network access.

What happens if you run 
`curl https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml`

Does it work?

**Comment by shvamabps** [comment]
![Screenshot (2)](https://user-images.githubusercontent.com/37547983/206113552-bb0280dd-70e3-4ca4-9437-aa007a72e8af.png)
It is still the same. get's no response.
Is it because there is no directory stable in the main repository? @kostis-codefresh

**Comment by shvamabps** [comment]
@kostis-codefresh 
If I explore the repository, the `install.yaml` file is not in the stable directory as there is no such directory in the first place. The file is this path `https://github.com/argoproj/argo-cd/blob/master/manifests/install.yaml`. I couldn't find any directory.

**Reply by kostis-codefresh** [reply]
Stable is not a directory. It is a git tag https://github.com/argoproj/argo-cd/tree/stable

Anyway, it seems that you have a network issue and not an argocd problem. Make the curl command work first and then the installation will also work.

**Reply by shvamabps** [reply]
Yeah it was network issue. Couldn't figure out why it was like that but it worked when I switched it to my mobile network.
